apps/desktop/src/components/ChromeHeader.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	import SyncButton from '$components/SyncButton.svelte';
+	import { platformName } from '$lib/platform/platform';
+	import { Project } from '$lib/project/project';
+	import { getContext } from '@gitbutler/shared/context';
+	import Button from '@gitbutler/ui/Button.svelte';
+	import { env } from '$env/dynamic/public';
+
+	const project = getContext(Project);
+</script>
+
+<nav>
+	<div class="left">
+		{#if platformName === 'macos' || env.PUBLIC_TESTING}
```

Comment:
How come you added `env.PUBLIC_TESTING`?

Reply:
We used it in other places with the stoplights, not sure if you mac folks used this. But if not I can just drop it 👍

Reply:
I think we can drop it!